deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml

```
@@ -33,6 +33,12 @@ data:
       - MetricsClientSendFailingSRE
       - UpgradeNodeScalingFailedSRE
       - UpgradeClusterCheckFailedSRE
+    extDependencyAvailabilityChecks:
+      http:
+        timeout: 10
+        urls:
+          - https://quay.io/health
+          - https://api.openshift.com
```

This could better be represented as 
```suggestion
          - ${OCM_BASE_URL}
```

Staging MUOs should care about the staging API health, not Prod (etc..)